### PR TITLE
Figma Code Connect + Docs

### PR DESCRIPTION
**Pull request by slhinyc**
# v2.3.0
- Documentation and Figma Code Connect updates (no visible changes to components)
  - Set up Figma Code Connect for some components
  - Add Cypress testing docs to several form-related components
  - Add separate `TS Form` code example panel for components that can be rendered with `ts_form_for`
  - Update string syntax in Slim code examples to use boolean (e.g. `open=true` instead of `open='true'`)
  - Add code example for `sl-drawer` opened from `sl-dropdown` with button trigger
- Minor component styling updates
  - Update `sl-divider` default color
  - Update size of close icon (x) in `sl-alert`
  - Update `sl-tag` font weight to medium
  - Update spacing and hover state for `sl-tab-group` and `sl-tab` to match ViewComponent Tab

---
🔗 [Review app](https://shoelace-ft4dp64y2-teamshares.vercel.app/)

### CODE REVIEW

File: docs/assets/scripts/code-previews.js [hunk 1]
```diff
@@ -132,6 +139,10 @@
       // Show Slim
       setFlavor('slim');
       toggleSource(codeBlock, true);
+    } else if (button?.classList.contains('code-preview__button--simple-form')) {
+      // Show Slim
```
Comment (kdonovan) [inline docs/assets/scripts/code-previews.js]:
```suggestion
      // Show SimpleForm
```

File: docs/pages/components/checkbox-group.md [hunk 2]
```diff
@@ -6,17 +6,87 @@ layout: component
...
+
+  To test `sl-checkbox-group` implemented with `ts_form_for`, add `data-test-id` to `wrapper_html`. To test checkbox items in the group, add `data-test-id` to each item in the collection array:
+
+  ```js
+      = ts_form_for ... do |f|
+        = f.input :input_name,
+          as: :check_boxes,
+          label: "Checkbox group text",
+          collection: [
+            ["Option 1", :option-1, data: { test_id: "item-test-1" }],
```
Comment (kdonovan) [inline docs/pages/components/checkbox-group.md:58]: I forgot we could do this -- that's a _great_ tip!

**Review by kdonovan — APPROVED** [review]
That was... a lot 😅 I assume it's been working for you, though, and nothing jumps out as problematic!

File: docs/assets/scripts/code-previews.js (hunk 1 above)
Comment (slhinyc) [reply docs/assets/scripts/code-previews.js]: Oops! Copy-pasta mistake -- TY for catching it! 🙏

**Comment by slhinyc** [comment]
> That was... a lot 😅 I assume it's been working for you, though, and nothing jumps out as problematic!

Yes... sorry so much at once 😅 . I did check everything, including pulling into cash-account to check the style-only `sl-tab-group` change & checking in Browserstack & all works as expected.

**Pull request merged**